src/presentation/controller/registration/Routing.ts

```diff
@@ -11,11 +11,14 @@ import PageType from "../PageType";
 const START_URL = `${BASE_URL}/start`;
 export const WHICH_TYPE_TEMPLATE = RegistrationPageType.whichType;
 export const NAME_TEMPLATE = RegistrationPageType.name;
+export const GENERAL_PARTNERS_TEMPLATE = RegistrationPageType.generalPartners;
 export const NEXT_TEMPLATE = RegistrationPageType.next;
 
 export const WHICH_TYPE_URL = `${BASE_URL}/${WHICH_TYPE_TEMPLATE}`;
 export const NAME_URL = `${BASE_URL}/${NAME_TEMPLATE}`;
 
+export const GENERAL_PARTNERS_URL = `${BASE_URL}/transaction/${TRANSACTION_ID}/submission/${SUBMISSION_ID}/${GENERAL_PARTNERS_TEMPLATE}`;
```

Comment:
we have a bit of duplication now with the url paths, can you make ${BASE_URL}/transaction/${TRANSACTION_ID}/submission/${SUBMISSION_ID}
into a BASE_WITH_IDS_URL ?

Reply:
Promoted this to a constant.